src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssPropertiesDotComa = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.length > 0);
+
+  const cssPropertiesDoubleDot = cssPropertiesDotComa
+    .map((item) => item.split(':'))
+    .map((splittedCssProperty) =>
+      // eslint-disable-next-line prettier/prettier
+      splittedCssProperty.map((propertyPart) => propertyPart.trim()));
```

Comment:
[CODE STYLE] - Avoid variable namings like `cssPropertiesDotComa` and `cssPropertiesDoubleDot` as they are not very descriptive and can decrease code readability. Consider more meaningful names that describe the content of the variables.